ucp/core.py

```diff
@@ -232,6 +235,7 @@ class ApplicationContext:
 
     def __init__(self, config_dict={}, blocking_progress_mode=None):
         self.progress_tasks = []
+        self.progress_tasks = []
```

Comment:
```suggestion
```

Reply:
Whoops. For some reason that has creeped in more than once.